### DISCUSSION TITLE
React hook form with Email js ,not getting an empty value on my gmail

### DISCUSSION THREAD

**Discussion by bennetho** [react-hook-form/react-hook-form#2619] [Q&A]
I tried to use Email js with react hook form but am getting an empty message  on my gmail. Am not getting the value however the value does appeared on console.

Please help

**Comment by bluebill1049** [comment]
can you share your form in a codesandbox?

**Comment by bennetho** [comment]
https://codesandbox.io/s/busy-northcutt-phuxt?file=/src/styles.css

**Reply by bluebill1049** [reply]
check your CSB, it's empty...

**Reply by bennetho** [reply]
It’s done

**Reply by bennetho** [reply]
Check it please

**Reply by bluebill1049** [reply]
<img width="690" alt="Screen Shot 2020-08-16 at 10 41 36 pm" src="https://user-images.githubusercontent.com/10513364/90334534-a6756a00-e011-11ea-973d-f16afff55186.png">
looks fine here.

**Reply by bennetho** [reply]
No was empty

**Reply by bluebill1049** [reply]
that's not an issue with this lib then, it's collect data correctly for you.

**Reply by bennetho** [reply]
How can I fix that.. it works well class state component

**Reply by bluebill1049** [reply]
i would try to isolate the problem just create an object and see if your email send works.

**Reply by bluebill1049** [reply]
this has nothing to do with class or function component.

**Reply by bennetho** [reply]
Ok thanks

**Comment by bennetho** [comment]
![9D6CEFEF-9DA8-4BEE-9332-6606A1143955](https://user-images.githubusercontent.com/64750931/90334567-7654a980-dfc6-11ea-8f92-92c4d2901d77.png)

**Reply by ahmedamine12** [reply]
is there any new solution to this problem?

**Reply by leapful** [reply]
@ahmedamine12 : Since react-hook-form is being used in this discussion is v6 while the latest version is v7, it's already obsoleted .  Could you please open a new discussion and provide a codesandbox url to re-produce the problem ?
